lib/request-base.js

```diff
@@ -3,6 +3,19 @@
  */
 
 /**
+ * Root reference for environment-specific globals.
+ */
+
+var root;
+if (typeof window !== 'undefined') { // Browser window
+  root = window;
+} else if (typeof self !== 'undefined') { // Web Worker
+  root = self;
+} else { // Other environments (node or other)
+  root = {FormData: require('form-data')};
```

Comment:
How does it behave when compiled for the browser? Will it cause an unused module to be bundled?